pkgs/games/openra/mods.nix

```diff
@@ -0,0 +1,246 @@
+{ buildOpenRAMod, fetchFromGitHub, extraPostFetch }:
+
+{
```

Comment:
Probably `recurseIntoAttrs` is needed here

Reply:
This is not telling me much, could you elaborate?

Reply:
`recurseIntoAttrs` is needed for `nix-env` and Hydra to find the derivations